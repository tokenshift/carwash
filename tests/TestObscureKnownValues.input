This is a test. Don't let anyone see this: P@ssw0rd!
There's a bunch of numbers 5432123454321 as well
And then some numbers 12345 and P@ssw0rd! on the same line
