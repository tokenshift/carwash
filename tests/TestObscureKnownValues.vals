P@ssw0rd!
12345
