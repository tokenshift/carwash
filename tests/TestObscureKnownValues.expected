This is a test. Don't let anyone see this: ********
There's a bunch of numbers 5432********4321 as well
And then some numbers ******** and ******** on the same line
